src/adapter.js

```diff
@@ -185,6 +185,71 @@ function attachmentNameError(name) {
   return false;
 }
 
+function cacheUpdateRequired(api, cache, designDocName, callback) {
+  cache.seq = cache.seq || 0;
+  var changesOpts = {
+    doc_ids: [ '_design/' + designDocName ],
+    limit: 1,
+    since: cache.seq
+  };
+  api.changes(changesOpts)
+    .then(function(res) {
```

Comment:
Small nit, general style is that as long as they can fit we do

```api.changes(changesOpts).then(function (res) { 
   // rest of code here
});
```

Theres a few other examples of this down below